src/js/api/brand.js

```diff
@@ -624,6 +661,29 @@ ripe.Ripe.prototype._runLogicOptions = function(options = {}) {
     });
 };
 
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype._getLogicOptions = function(options = {}) {

```

Comment:
Isn't `method` missing? See: https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/controllers/api/brand.py#L227

And since we're working on this, is https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/controllers/api/brand.py#L243 already in RIPE SDK?

Reply:
I din't implement the method name because `runLogic` (which is the `logic_method` endpoint) already does that. The getLogic returns the full script.

Reply:
Sure but if it exists in the RIPE Core it must exist here too, there's no reason for the API client to restrict what can be done with the API it represents.

Reply:
Regarding the second point I can see the answer is `yes`

Reply:
@gcandal While I was implementing this I noticed a problem in the logic endpoint. It is impossible (i think) to pass the `ctx` as arguments for the `method`, since it is an object.

Reply:
It must work, because the SDK calls `on_part` a lot. See onPart/onPartP/_onPartOptions/etc

Reply:
@gcandal The problem is that those endpoints are POST, so they send the arguments as body, however the logic endpoint that `getLogic` calls is GET

Reply:
You're right!

Reply:
That's not a problem in practice though, is it?

Reply:
The problem resides in passing a nested object through the URL...i can stringify it, but on the core side it should convert it back to object. Its this, or I'm forgetting something super basic :sweat_smile: